### PR TITLE
Removed corpnet links. Updated links to aspnet. Updated section on PDBs. Fixed typos.

### PR DESCRIPTION
**Pull request by DanielMeixner**
Removed corpnet links. Updated links to aspnet. Updated section on PDBs. Fixed typos.

### CODE REVIEW

File: debugger.md [hunk 1]
```diff
@@ -1,60 +1,48 @@
...
 Thank you for taking the time to try out our bits. 
 This page gives you detailed instructions on how to get your system running. 
-It will be updated on a regulary basis to always reflect the latest behaviour. Now let's get started.​
 
 ####Your Feedback​
-
 Please place your feedback [here](https://github.com/OmniSharp/omnisharp-vscode/issues). 
 
 ####Prerequisites / Known Issues / Notes
 * X64 only
```
Comment (gregg-miskelly) [inline debugger.md]: I would suggest we lead with the fact that this requires a recent .NET CLI bits. Something like:
* Requires .NET Core rc3-23829 or newer. These are daily builds of .NET Core that can be downloaded from the https://www.myget.org/F/dotnet-core/api/v3/index.json nuget feed. These are really daily builds of .NET Core and associated tools. So please continue if you want to dig in and have fun playing with new bits. But don't expect to go to production with these things.

File: debugger.md [hunk 2]
```diff
@@ -1,60 +1,48 @@
...
-* Currently NOT working on Windows
 * In case you get this error on F5: *"No task runner configured -  Tasks.json could not be found"*, see the 'once for each project' section below. 
 * If you don’t have mono installed you won't get IntelliSense. 
-* **Hint:** Setup for Debugging CLR on OSX changed fundamentally end of February 2016. For existing projects please make sure to update your launch.json file according to these steps.​
 
 ###First Time setup
 1. Get Visual Studio Code
-    * Install Visual Studio Code. Pick the latest insider version from here: https://code.visualstudio.com/insiders
-    * **Hint:** While we are in a pre-release phase vscode might ask you to install a C# extension from the marketplace. DO NOT INSTALL IT! This extension doesn't contain debugger bits yet and might bring your system in a weird state.   
+ * Install Visual Studio Code (VSC). Pick the latest VSC version from here: https://code.visualstudio.com/insiders Make sure it is at least 0.10.10.
```
Comment (gregg-miskelly) [inline debugger.md]: Remove the 'insiders' part now that the stable version is new enough?

File: debugger.md [hunk 3]
```diff
@@ -1,60 +1,48 @@
...
-* **Hint:** Setup for Debugging CLR on OSX changed fundamentally end of February 2016. For existing projects please make sure to update your launch.json file according to these steps.​
 
 ###First Time setup
 1. Get Visual Studio Code
-    * Install Visual Studio Code. Pick the latest insider version from here: https://code.visualstudio.com/insiders
-    * **Hint:** While we are in a pre-release phase vscode might ask you to install a C# extension from the marketplace. DO NOT INSTALL IT! This extension doesn't contain debugger bits yet and might bring your system in a weird state.   
+ * Install Visual Studio Code (VSC). Pick the latest VSC version from here: https://code.visualstudio.com/insiders Make sure it is at least 0.10.10.       
 2. Install Dotnet CLI
  * Install Dotnet CLI following the instructions here:  http://dotnet.github.io/getting-started  
  * **Hint for Mac**: Dotnet CLI requires openSSL to work. Don't forget this! Execute: `brew install openssl`
```
Comment (gregg-miskelly) [inline debugger.md]: On Windows we need to tell people to install something newer than what is on the web site.

File: debugger.md [hunk 4]
```diff
@@ -1,60 +1,48 @@
...
-  * Or you can open the command pallette in VSC (F1) and run the command *"Debugger: Add tasks.json"* (Make sure you have installed Mono to be able to do so, see above. If you don't have Mono installed you might get an error saying *Omnisharp not running*.)
-  * Or if you just want debugging, you can comment out 'preCompileTask' field​
-  * Modify your project.json to reference "NETStandard.Library":"1.0.0-rc3-*"
-   * Currently dotnet new creates a project.json that references a version that's too old. You should also get a hint to modify this.
+  * Or you can open the command palette in VSC (F1) and run the command *"Debugger: Add tasks.json"* (Make sure you have installed Mono to be able to do so, see above. If you don't have Mono installed you might get an error saying *Omnisharp not running*.)
+  * Or if you just want debugging, you can comment out 'preCompileTask' field​ in launch.json.
+* Modify your project.json to reference "NETStandard.Library":"1.0.0-rc3-*"
+   * Currently *dotnet new* creates a project.json that references a version that's too old. You should also get a hint to modify this.
    * Afterwards run *dotnet restore*.
```
Comment (gregg-miskelly) [inline debugger.md]: Need Windows only step of enabling portable PDBs

File: debugger.md [hunk 5]
```diff
@@ -1,60 +1,48 @@
...
 * In case you get a restore error due to lack of a NuGet.Config file, just create this file in the root directory of your project. You can find a sample [here](https://github.com/Microsoft/MIEngine/blob/abeebec39221c654bd69a0d2bcadca6a4a0d0392/tools/InstallToVSCode/CLRDependencies/NuGet.Config). 
 
 ####Debugging Code compiled on another computer
-* The target app binaries must be built with Portable PDBs for symbol loading to work. 
-* If the target binary is built on Linux / OSX, dotnet CLI will produce portable pdbs by default so no action is necessary. 
-* If the target binary is built on windows, the pdb needs to be converted: [How to convert Windows PDBs to Portable](https://microsoft.sharepoint.com/teams/DD_VSPlat/Diagnostics/_layouts/15/WopiFrame.aspx?sourcedoc={872c5298-6f17-4960-a5a0-acc4f215e730}&action=edit&wd=target%28%2F%2FMDD%2FXPlat%20CLR.one%7C1b640b59-6617-4452-b360-c24e9d5cad48%2FHow%20to%20convert%20Windows%20PDBs%20to%20Portable%7C49651c0c-f5a0-4d77-9478-9a233f0bf345%2F%29)
+* The target app binaries must be built with Portable PDBs for symbol loading to work. On Window Portable PDBs are only created when using dotnet cli for compilation.
```
Comment (gregg-miskelly) [inline debugger.md]: This isn't correct. You can get portable PDBs with msbuild, and they aren't generated by default with .NET CLI. See the email I sent you.

**Comment by gregg-miskelly** [comment]
I have finished reviewing.

**Comment by DustinCampbell** [comment]
@DanielMeixner, once you've got this ready, could you squash commits. Or, could you rebase on top of dev rather than merging?

**Comment by DustinCampbell** [comment]
This was committed with https://github.com/OmniSharp/omnisharp-vscode/commit/ea35c67ec4a671bb72e7ec4d69bf53a267e21122

**Pull request closed without merging**